packages/vue-vanilla/src/controls/InputControlRenderer.vue

```diff
@@ -5,40 +5,35 @@
     :is-focused="isFocused"
     :applied-options="appliedOptions"
   >
-    <input
-      :id="control.id + '-input'"
-      :class="styles.control.input"
-      :value="control.data"
-      :disabled="!control.enabled"
-      :autofocus="appliedOptions.focus"
-      :placeholder="appliedOptions.placeholder"
-      @change="onChange"
-      @focus="isFocused = true"
-      @blur="isFocused = false"
+    <DispatchCell
+      v-bind="control"
+      :id="controlWrapper.id"
+      :handle-change="handleChange"
```

Comment:
Similar to React Vanilla ([see here](https://github.com/eclipsesource/jsonforms/blob/24f4e5f6280263a4b8e1263aa2a302613b546272/packages/vanilla-renderers/src/controls/InputControl.tsx#L108-L114)) we should only hand over the minimum and not bind all of `control` here. The cell bindings will take care of the rest